Makefile

```diff
@@ -2,173 +2,95 @@ ifneq (,)
 .error This Makefile requires GNU Make.
 endif
 
-.PHONY: help gen lint test _gen-main _gen-examples _gen-modules _lint_files _lint_fmt _pull-tf _pull-tf-docs
-
+# -------------------------------------------------------------------------------------------------
+# Default configuration
+# -------------------------------------------------------------------------------------------------
+.PHONY: help lint lint-files terraform-docs terraform-fmt _pull-tf _pull-tfdocs
 CURRENT_DIR     = $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
-TF_EXAMPLES     = $(sort $(dir $(wildcard $(CURRENT_DIR)examples/*/)))
-TF_MODULES      = $(sort $(dir $(wildcard $(CURRENT_DIR)modules/*/)))
 
-TF_VERSION      = light
-TF_DOCS_VERSION = 0.6.0
+
+# -------------------------------------------------------------------------------------------------
+# Docker image versions
+# -------------------------------------------------------------------------------------------------
+TF_VERSION     = 0.13.7
```

Comment:
Should we use 0.12.x to lint and fmt ?

Reply:
The module is formally `0.12` compatible, but that does not mean we should limit it.